graylog2-web-interface/package.json

```diff
@@ -60,6 +60,7 @@
     "core-js": "3",
     "crossfilter": "1.3.x",
     "dompurify": "^2.0.11",
+    "glob-parent": "6.0.2",
```

Comment:
Just because I am curious, why are we now listing `glob-parent` as a dependency?

Reply:
That is actually a mistake. It should actually be bumped in the lockfile only, I missed this in `git diff`. Thanks!

Reply:
Alright!